db/schema.rb

```diff
@@ -55,6 +55,18 @@
     t.index ["user_id"], name: "index_authorizations_on_user_id"
   end
 
+  create_table "backup_code_configurations", force: :cascade do |t|
+    t.integer "user_id", null: false
+    t.string "encrypted_code", default: "", null: false
+    t.string "code_fingerprint", default: "", null: false
+    t.boolean "used", default: false
+    t.datetime "used_at"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["code_fingerprint"], name: "index_backup_code_configurations_on_code"
```

Comment:
Rather than two separate indexes we want a compound index: ["user_id", "code_fingerprint"]

Reply:
Fixed